sdk/python/feast/client.py

```diff
@@ -514,6 +514,19 @@ def list_entities(self) -> Dict[str, Entity]:
                 entities_dict[entity.name] = entity
         return entities_dict
 
+    def list_features(self) -> Dict[str, Feature]:
+        """
```

Comment:
I think this will cause a conflict for features that have the same name in different projects.

Reply:
I saw there is a function called `list_entities` therefore I wonder why the `list_features` function is not there.  Do you have a name suggestion for this function?

Oh yeah, was creating the `list_features` function to see the labels on features, because as I saw the signature of `list_feature_set` function, it only returns with the format `project/name:version` . It doesn't explicit enough to show the complete feature set, including the features and labels

Reply:
Can we remove this for now?